src/safeStringify.ts

```diff
@@ -1,7 +1,5 @@
 import { DumpOptions, safeDump } from '@stoplight/yaml-ast-parser';
 
 export const safeStringify = (value: any, options?: DumpOptions): string => {
-  if (!value || typeof value === 'string') return value;
-
   return safeDump(value, options);
```

Comment:
Hmm, what if we just did

```suggestion
  return typeof value === 'string' ? value : safeDump(value, options);
```

Reply:
Going to leave this up to you and @marbemac though, but I would rather be consistent and either fix the tests in `json-shcema-ref-parser`.

Reply:
Yeah, I'm fine with updating tests in json-schema-ref-parser.